app/controllers/works_controller.rb

```diff
@@ -18,61 +18,66 @@ class WorksController < ApplicationController
   before_action :set_instance_variables, only: [:new, :create, :edit, :update, :manage_chapters, :preview, :show, :navigate, :import]
   before_action :set_instance_variables_tags, only: [:edit_tags, :update_tags, :preview_tags]
 
-  before_action :clean_work_search_params, only: [:search, :index, :collected]
-
   cache_sweeper :collection_sweeper
   cache_sweeper :feed_sweeper
 
   # we want to extract the countable params from work_search and move them into their fields
   def clean_work_search_params
-    if params[:work_search].present? && params[:work_search][:query].present?
+    #https://aaronlasseigne.com/2014/07/20/know-ruby-clone-and-dup/
+    clean_params = work_search_params&.dup
+    clean_params[:query] = clean_params[:query].dup  unless clean_params[:query].nil?
+    clean_params[:term] = clean_params[:term].dup unless clean_params[:term].nil?
+    clean_params[:sort_column] = clean_params[:sort_column].dup unless clean_params[:sort_column].nil?
+    clean_params[:sort_direction] = clean_params[:sort_direction].dup unless clean_params[:sort_direction].nil?
+
+    if clean_params.present? && clean_params[:query].present?
       # swap in gt/lt for ease of matching; swap them back out for safety at the end
-      params[:work_search][:query].gsub!('&gt;', '>')
-      params[:work_search][:query].gsub!('&lt;', '<')
+      clean_params[:query].gsub!('&gt;', '>')
+      clean_params[:query].gsub!('&lt;', '<')
 
       # extract countable params
       %w(word kudo comment bookmark hit).each do |term|
-        next unless params[:work_search][:query].gsub!(/#{term}s?\s*(?:\_?count)?\s*:?\s*((?:<|>|=|:)\s*\d+(?:\-\d+)?)/i, '')
+        next unless clean_params[:query].gsub!(/#{term}s?\s*(?:\_?count)?\s*:?\s*((?:<|>|=|:)\s*\d+(?:\-\d+)?)/i, '')
         # pluralize, add _count, convert to symbol
         term = term.pluralize unless term == 'word'
         term += '_count' unless term == 'hits'
         term = term.to_sym
 
         value = Regexp.last_match(1).gsub(/^(\:|\=)/, '') # get rid of : and =
         # don't overwrite if submitting from advanced search?
-        params[:work_search][term] = value unless params[:work_search][term].present?
+        clean_params[term] = value unless clean_params[term].present?
```

Comment:
Perhaps a silly question! This line uses `clean_params[term]`, but on line 29, it's `clean_params[:term]`. Is that intentional? And if it's intentional, do we still need line 29? I can't see where `clean_params[:term]` is being used.